static/app/views/insights/common/components/modulePageFilterBar.tsx

```diff
@@ -104,7 +104,7 @@ export function ModulePageFilterBar({
       >
         {/* TODO: Placing a DIV here is a hack, it allows the tooltip to close and the ProjectPageFilter to close at the same time,
           otherwise two clicks are required because of some rerendering/event propogation issues into the children */}
-        <div style={{width: '100px', position: 'absolute', height: '100%'}} />
+        <div style={{position: 'absolute', width: '100px', height: '36px'}} />
```

Comment:
just wondering where `36px` comes from and if this will be the same value between old ui / chonk ?

Reply:
great callout, that's the button height (36 in old UI, 38 in chonk which is good enough for this usage.) i'll add this to the comment

Reply:
so this will still work in chonk if we set it to `36px` even though in chonk it’s `38px` ?